plat/kvm/arm/exceptions.S

```diff
@@ -113,20 +113,72 @@
 	 */
 	mrs	x0, tpidr_el1
 	ldr	x0, [x0, #LCPU_AUXSP_OFFSET]
+	sub	x0, x0, #UKARCH_AUXSPCB_SIZE
+	ldr	x0, [x0, #UKARCH_AUXSPCB_OFFSETOF_CURR_FP]
+
+	/**
+	 * Current stack pointer now points to the current frame pointer
+	 * within the auxiliary stack. Subtract UKARCH_EXECENV_END_ALIGN
+	 * to make room for the 8-byte auxsp pointer, since the layout is
+	 * struct uk_syscall_ctx {
+	 *	struct ukarch_execenv execenv;
+	 *	__uptr auxsp;  <-- make room for these
+	 *     ..... leftover room because of alignment subtraction ....
```

Comment:
See similar comments as given for `syscall.S`

Reply:
> Hmm, I intentionally said "leftover" because it is an unfortunate consequence of having to create a 64 byte slot only to store an 8 byte value just because of alignment constraints. Padding sounds like it's something that we intentionally add? I don't know 🤔

Ditto